milabench/multi.py

```diff
@@ -4,12 +4,12 @@
 
 from voir.instruments.gpu import get_gpu_info
 
+import milabench.executors as exec
```

Comment:
`exec` is a Python built-in func : https://docs.python.org/3/library/functions.html#exec

Might be best to use some other name